distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -493,25 +492,6 @@ public void testBasicRestAccess() throws Exception {
         getSecurityPolicy().waitForGuestAuthReady(url);
     }
 
-    @Test(expected = NoHttpResponseException.class)
-    public void testBasicAuthOverHttp() throws Exception {
```

Comment:
The test is basically just trying HTTP against the HTTPS port, which is always going to fail.